bedrock/products/urls.py

```diff
@@ -41,4 +31,12 @@
         views.resource_center_article_view,
         name="products.vpn.resource-center.article",
     ),
+    # Issue #12937 - updating VPN subnav
+    redirect(r"^vpn/more/what-is-an-ip-address/$", "/products/vpn/resource-center/what-is-an-ip-address/", permanent=True),
+    redirect(r"^vpn/more/what-is-a-vpn$", "/products/vpn/resource-center/what-is-a-vpn/", permanent=True),
+    redirect(r"^vpn/more/vpn-or-proxy/$", "/products/vpn/resource-center/the-difference-between-a-vpn-and-a-web-proxy/", permanent=True),
+    redirect(r"^vpn/more/when-to-use-a-vpn/$", "/products/vpn/resource-center/5-reasons-you-should-use-a-vpn/", permanent=True),
+    redirect(r"^vpn/more/why-mozilla-vpn/$", "products.vpn.landing", permanent=True),
+    redirect(r"^vpn/more/do-i-need-a-vpn/$", "/products/vpn/resource-center/do-you-need-a-vpn-at-home-here-are-5-reasons-you-might/", permanent=True),
+    redirect(r"^vpn/more/what-is-a-vpn-v2/$", "/products/vpn/resource-center/what-is-a-vpn/", permanent=True),
```

Comment:
Suggestion (non blocking): Is it worth making a separate `redirects.py` file for these, like we do for other bedrock apps such as firefox and mozorg?

Reply:
I think you might also be able to omit `permanent=True`, since this seems to be the [default](https://github.com/mozilla/bedrock/blob/main/bedrock/redirects/util.py#L116)?